assimp 4.0.0 support

https://github.com/assimp/assimp/releases/tag/v4.0.0 was just released, but pioneer's configure rejects it:
```
configure: error: "assimp version >= 3.2 not found. Get it from https://assimp.sourceforge.net/"
```

https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/4890/version=sierra/testReport/junit/brew-test-bot/sierra/install_pioneer/
https://github.com/Homebrew/homebrew-core/pull/15763

--- a/configure.ac
+++ b/configure.ac
@@ -302,7 +302,9 @@ if test "x$cross_compiling" = "xno" ; then
 	AC_RUN_IFELSE([AC_LANG_PROGRAM([[#include <assimp/defs.h>
 	                                 #include <assimp/version.h>]],
 	                            [[do {
-	                                if (aiGetVersionMajor() >= 3 && aiGetVersionMinor() >= 2)
+	                                if (aiGetVersionMajor() > 3)
+	                                    return 0;
+	                                else if (aiGetVersionMajor() == 3 && aiGetVersionMinor() >= 2)
 	                                    return 0;
 	                                return 1;
 	                              } while(0)]])],

